python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -151,6 +151,9 @@ class CoaddAnalysisConfig(Config):
     fluxToPlotList = ListField(dtype=str, default=["base_GaussianFlux", "base_CircularApertureFlux_12_0",
                                                    "ext_photometryKron_KronFlux", "modelfit_CModel"],
                                doc="List of fluxes to plot: mag(flux)-mag(base_PsfFlux) vs mag(fluxColumn)")
+    gaapFluxList = ListField(dtype=str, default=["ext_gaap_GaapFlux_1_15x_Optimal",
```

Comment:
I don't quite see the need to have this as a separate field. Sure, during the initial phases may be not all GAaP quantities that should have been there in the schema were there, but now that it is straightened out, this could be with the above list.